bigtable-hbase-parent/bigtable-hbase-integration-tests-2.x/src/test/java/com/google/cloud/bigtable/hbase/test_env/BigtableEnv.java

```diff
@@ -98,4 +111,9 @@ protected void teardown() {
   public Connection createConnection() throws IOException {
     return ConnectionFactory.createConnection(configuration);
   }
+  
+  public CompletableFuture<AsyncConnection> createAsyncConnection() throws IOException {
```

Comment:
Why not return `AsyncConnection` instead of `CompletableFuture<AsyncConnection>`?

Reply:
This would provide a way to access the future (CF) object if in case we need during tests. I do not yet have a concrete case where this might be required. I am OK to change it to AsyncConnection if you prefer.